README.md

```diff
@@ -1,19 +1,67 @@
 NuCypher
 ========
+*A proxy re-encryption network to empower privacy in decentralized systems*
+
+- Website https://www.nucypher.com/
+- GitHub https://www.gihub.com/nucypher/nucypher
+
 
 .. image:: https://circleci.com/gh/nucypher/nucypher/tree/master.svg?style=svg
     :target: https://circleci.com/gh/nucypher/nucypher/tree/master
 
+.. image:: https://coveralls.io/repos/github/nucypher/nucypher/badge.svg?branch=master
+    :target: https://coveralls.io/github/nucypher/nucypher?branch=master
+
+
+
+NuCypher Proxy Re-encryption Network
+-------------------------------------
+
+1. Alice, the data owner, encrypts data with her public key and uploads it to IPFS, Swarm, S3, or any supported storage layer.
```

Comment:
I think, "supported" could be misleading here. It could be literally _any_ storage layer. We may make it convenient to use some so that you don't call NuCypher explicitly, but we don't limit ourselves to only "supported" layers

Reply:
Agreed - On a higher level - I'm not entirely sure what belongs on this front page quite yet.  Any Ideas?

Reply:
I'd just remove the word "supported" and call it a day

Reply:
:heavy_check_mark: